packages/app-store/routing-forms/trpc/report.handler.ts

```diff
@@ -123,12 +129,13 @@ function presenter(args: {
   const formatDate = makeFormatDate(ctx.user.locale, ctx.user.timeZone);
   return {
     nextCursor,
-    headers: [...headers, "Routed To", "Booked At", "Submitted At"],
+    headers: [...headers, "Routed To", "Assignment Reason", "Booked At", "Submitted At"],
     responses: responses.map((r, i) => {
       const currentRow = rows[i];
       return [
         ...r,
         currentRow.routedToBooking?.user?.email || "",
+        currentRow.routedToBooking?.assignmentReason[0].reasonString || "",
```

Comment:
By recording the string in the database, we don't have to do any processing which increases performance.